esdoc.json

```diff
@@ -38,6 +38,10 @@
       "./doc/tutorial/basics.md",
       "./doc/tutorial/applications.md",
       "./doc/external-references.md"
+    ],
+    "usage": [
+      "./doc/writing-marble-tests.md",
+      "./doc/operator-creation.md"
```

Comment:
I agree with @staltz ,  put them in the tutorial section would be better. I will make the change later.